src/app/globals.css

```diff
@@ -1,35 +1,61 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
 :root {
-  
 }
 
 * {
-  box-sizing: border-box;
-  padding: 0;
-  margin: 0;
+	box-sizing: border-box;
+	padding: 0;
+	margin: 0;
 }
 
 html,
 body {
-  max-width: 100vw;
-  overflow-x: hidden;
+	max-width: 100vw;
+	overflow-x: hidden;
+}
+
+body {
+	background-color: rgba(128, 93, 190, 0.07);
+	height: 100%;
+}
+
+.container {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	min-height: 37.5rem;
+	height: 95dvh;
+	@apply max-w-none;
 }
 
-body{
-  background-color: rgba(128, 93, 190, 0.07);
-  height: 100%;
+main {
+	flex: 1;
+	background-color: #ffffff;
+	border-radius: 20px;
+	margin: 0 50px;
 }
 
-.container{
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  min-height: 37.5rem;
-  height: 95dvh;
+/* width */
+::-webkit-scrollbar {
+	width: 5px;
 }
 
-main{
-  flex:1;
-  background-color: #ffffff;
-  border-radius: 20px;
-  margin: 0 50px;
-}
\ No newline at end of file
+/* Track */
```

Comment:
Why was this added?

Reply:
They are for the inner scroll within the browse credits page. I went ahead and commented and added the additional styling being used in our current main branch.